test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -301,37 +301,6 @@ Describe "Import-Module should be case insensitive" -Tags 'CI' {
     }
 }
 
-Describe "Workflow .Xaml module is not supported in PSCore" -tags "Feature" {
```

Comment:
Should we keep a single negative test case?  What error will get returned?

Reply:
Before the fix:
```
ipmo : Cannot load the workflow module 'XamlTest.xaml'. Workflow is not supported in PowerShell 6+.
At line:1 char:1
+ ipmo C:\temp\1\XamlRootModule.psd1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [Import-Module], NotSupportedException
+ FullyQualifiedErrorId : Modules_WorkflowModuleNotSupported,Microsoft.PowerShell.Commands.ImportModuleCommand
```

After the fix (there was a bug - fixed in new commit):
```
ipmo : The module to process 'XamlTest.xaml', listed in field 'NestedModules' of module manifest '' was not processed. The extension '.xaml' is not a valid module extension. The supported module extensions are '.dll', '.ps1', '.psm1', '.psd1' and '.cdxml'. Correct the extension then try adding the file 'XamlTest.xaml' again.
At line:1 char:1
+ ipmo C:\temp\1\XamlRootModule.psd1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [Import-Module], PSInvalidOperationException
+ FullyQualifiedErrorId : Modules_InvalidModuleExtension,Microsoft.PowerShell.Commands.ImportModuleCommand
```

Reply:
It seems we do the same previously with conclusion that make no sense to create special information exception for every feature which we don't support now.